.evergreen/config.yml

```diff
@@ -24116,6 +24116,112 @@ tasks:
       AUTH: noauth
       LOADBALANCED: loadbalanced
       SSL: nossl
+- name: testazurekms-task
+  commands:
+  - func: fetch source
+  - command: shell.exec
+    params:
+      shell: bash
+      script: |-
+        set -o errexit
+        echo "Building test-azurekms ... begin"
+        pushd mongoc
+        ./.evergreen/compile-test-azurekms.sh
+        popd
+        echo "Building test-azurekms ... end"
+        echo "Copying files ... begin"
+        export AZUREKMS_RESOURCEGROUP=${testazurekms_resourcegroup}
+        export AZUREKMS_VMNAME=${AZUREKMS_VMNAME}
+        export AZUREKMS_PRIVATEKEYPATH=/tmp/testazurekms_privatekey
+        DRIVER_TOOLS=$(pwd)/drivers-evergreen-tools
```

Comment:
nit: If I understand this code correctly, this looks unnecessary if you use DRIVERS_TOOLS env variable

Reply:
got it